docs/getting-started/leftNav.md

```diff
@@ -1,12 +1,12 @@
 ## [Dashboards](/getting-started/registration-dashboard?tab=1)
 
-##### [iModel Registration Dashboard](/getting-started/registration-dashboard?tab=1)
+##### [My iModels](/getting-started/registration-dashboard?tab=1)
 
-##### [Application Registration Dashboard](/getting-started/registration-dashboard?tab=0)
+##### [My Applications](/getting-started/registration-dashboard?tab=0)
 
 ---
 
-## [Offline](./offline-quickstart.md)
+## [Offline (Quick Start)](./offline-quickstart.md)

```

Comment:
At some point should we link these to the Tutorials instead of these separate files?  That can be a different PR, just curious if we had plans to do it.

Reply:
Makes sense or Online; there is quite a but more info in develop-desktop-viewer.md. But I'm not sure it makes sense for Offline. "Quick Start" and develop-desktop-viewer are essentially the same thing.